Client/src/main/java/com/minecrafttas/tasbattle/TASBattle.java

```diff
@@ -0,0 +1,27 @@
+package com.minecrafttas.tasbattle;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import lombok.Getter;
+import net.fabricmc.api.ModInitializer;
+import net.fabricmc.loader.api.FabricLoader;
+
+public class TASBattle implements ModInitializer{
+
+	@Getter
+	private static boolean isDevEnvironment=false;
+	
+	public static final Logger LOGGER = LogManager.getLogger("TASBattle");
+	
+	public static final Tickratechanger tickratechanger = new Tickratechanger();
+	
+	public static final SpectatorManager spectatormanager = new SpectatorManager();
```

Comment:
TickrateChanger instead of Tickratechanger, then initialize TickrateChanger and SpectatorManager in onInitialize() instead of the static variable. Also no static in general, add singleton if you need it (see lotas once again)